phpunit.xml.dist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit bootstrap="vendor/autoload.php" colors="true">
+    <testsuites>
+        <testsuite name="Mxm">
+            <directory>tests</directory>
```

Comment:
Isn't this `tests/unit`? I know it probably doesn't make a difference, but that's the directory with the unit tests in.